contracts/connectors/loantoken/LoanTokenLogicProxy.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "./AdvancedTokenStorage.sol";
+import "../../openzeppelin/Initializable.sol";
+
+/**
+ * @title Loan Token Logic Proxy contract.
+ *
+ * @notice This contract contains the proxy functionality and it will query the logic target from LoanTokenLogicBeacon
+ * This contract will also has the pause/unpause functionality. The purpose of this pausability is so that we can pause/unpause from the loan token level.
+ *
+ */
+contract LoanTokenLogicProxy is AdvancedTokenStorage {
+	/**
+	 * @notice PLEASE DO NOT ADD ANY VARIABLES HERE UNLESS FOR SPESIFIC SLOT
+	 */
+
+	/// ------------- MUST BE THE SAME AS IN LoanToken CONTRACT -------------------
+	address public sovrynContractAddress;
+	address public wrbtcTokenAddress;
+	address public target_;
+	address public admin;
+	/// ------------- END MUST BE THE SAME AS IN LoanToken CONTRACT -------------------
+
+	/**
+	 * @notice PLEASE DO NOT ADD ANY VARIABLES HERE UNLESS FOR SPESIFIC SLOT (CONSTANT / IMMUTABLE)
+	 */
+
+	bytes32 internal constant LOAN_TOKEN_LOGIC_BEACON_ADDRESS_SLOT = keccak256("LOAN_TOKEN_LOGIC_BEACON_ADDRESS_SLOT");
+
+	modifier onlyAdmin() {
+		require(isOwner(), "LoanTokenLogicProxy:unauthorized");
+		_;
+	}
+
+	function initializeLoanTokenProxy(address _beaconAddress) public onlyAdmin {
```

Comment:
`initializeLoanTokenProxy` is identical to `setBeaconAddress`
i suggest to replace for constructor or just remove it and also replace all `initializeLoanTokenProxy` for `setBeaconAddress`

Reply:
Initialized function has been removed.
Expected to use the setBeaconAddress instead.